ex03.ex

```diff
@@ -149,7 +158,25 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  def won({:x,:x,:x,_3,_4,_5,_6,_7,_8}),do: :x
```

Comment:
### -1

This is definitely correct. But you could have simplified it using variables to

~~~ elixir
def won({ x,  x,  x, _4, _5, _6, _7, _8, _9}), do:  x
def won({_1, _2, _3,  x,  x,  x, _7, _8, _9}), do:  x
def won({_1, _2, _3, _4, _5, _6,  x,  x,  x}), do:  x
def won({ x, _2, _3,  x, _5, _6,  x, _8, _9}), do:  x
def won({_1,  x, _3, _4,  x, _6, _7,  x, _9}), do:  x
def won({_1, _2,  x, _4, _5,  x, _7, _8,  x}), do:  x
def won({ x, _2, _3, _4,  x, _6, _7, _8,  x}), do:  x
def won({_1, _2,  x, _4,  x, _6,  x, _8, _9}), do:  x
def won(_), do: false
~~~